pkg/k8s/client.go

```diff
@@ -135,6 +137,30 @@ func (a *Access) IsJobFinished(namespacedName types.NamespacedName) (finished bo
 	return finished, nil
 }
 
+func (a *Access) GetJob(namespacedName types.NamespacedName) *batchv1.Job {
+	job, err := a.Clientset.BatchV1().Jobs(namespacedName.Namespace).Get(
+		namespacedName.Name, metav1.GetOptions{})
+	if err != nil {
+		return nil
+	}
+	return job
+}
+
+func (a *Access) GetJobPods(namespacedName types.NamespacedName) (*corev1.PodList, error) {
```

Comment:
minor/cosmetics: It might be easier to find this function if it would reside in `job.go`, but I can live this being here too.